Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -270,7 +272,7 @@ struct _WebKitWebViewBasePrivate {
     RefPtr<WebPageProxy> pageProxy;
     IntSize viewSize { };
     bool shouldForwardNextKeyEvent { false };
-    bool shouldForwardNextWheelEvent { false };
+    Vector<GdkEvent*> shouldForwardNextWheelEvent;
```

Comment:
This needs to be renamed now, e.g. `wheelEventsToForward`.

Also, if it's meant to hold ownership, then it should be `Vector<GRefPtr<GdkEvent>>`. But if it's not meant to hold ownership, then what you have here is fine.

Reply:
Done